File: FizzBuzz.scala
@main
def fizzBuzz() = {
  for(i <- 1 to 100) {
    var s = ""
    if(i % 3 == 0)  s += "Fizz"
    if(i % 5 == 0)  s += "Buzz"
    if(s == "")     s += i
    println(s)
  }
}

File: FizzBuzz.py
# Shortest (ignoring whitespace)
for i in range(100):
    print("Fizz" * (i % 3 == 0) + "Buzz" * (i % 5 == 0) or i + 1)

# Without using the modulo operator
v = 810092048
for i in range(100):
    j = v & 3
    v = v >> 2 | j << 28
    w = (i + 1, "Fizz", "Buzz", "FizzBuzz")
    print(w[j])
